src/sentry/seer/endpoints/organization_page_web_vitals_summary.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import logging
+
+import orjson
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.api.endpoints.organization_trace import OrganizationTraceEndpoint
+from sentry.models.organization import Organization
+from sentry.seer.page_web_vitals_summary import get_page_web_vitals_summary
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationPageWebVitalsSummaryPermission(OrganizationPermission):
+    scope_map = {
+        "POST": ["org:read"],
+    }
+
+
+@region_silo_endpoint
+class OrganizationPageWebVitalsSummaryEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.PERFORMANCE
+    enforce_rate_limit = True
+    rate_limits = {
+        "POST": {
+            RateLimitCategory.IP: RateLimit(limit=10, window=60),
+            RateLimitCategory.USER: RateLimit(limit=10, window=60),
+            RateLimitCategory.ORGANIZATION: RateLimit(limit=30, window=60),
+        }
+    }
+
+    def has_feature(self, organization: Organization, request: Request) -> bool:
+        return features.has(
+            "organizations:performance-web-vitals-seer-suggestions",
+            organization,
+            actor=request.user,
+        )
+
+    def post(self, request: Request, organization: Organization) -> Response:
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        data: dict = orjson.loads(request.body) if request.body else {}
+        trace_ids = data.get("traceSlugs", None)
+        if not trace_ids:
+            return Response({"detail": "Missing traceSlugs parameter"}, status=400)
+
+        try:
+            trace_endpoint = OrganizationTraceEndpoint()
```

Comment:
if you want to use methods from this endpoint, you should consider making the methods static, or preferably, making these utility methods. having to initialize an endpoint to call methods is not an ideal pattern.